project_setup/scripts/terraform-to-secrets

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python
+
+"""Parse Terraform state and create GitHub secrets.
+
+Secrets are create for IAM access keys, and specially tagged resources.
+
+For each IAM access key two secrets will be created:
+ - AWS_ACCESS_KEY_ID
+ - AWS_SECRET_ACCESS_KEY
+
+If there is only one access key, the secrets will be named as above.  If more than one
+key exists in the Terraform state, the key's name will be appended to the secret's name.
```

Comment:
Isn't it the user's name (owner of the key) that is appended, not the key's name?

Reply:
Yup.  I was lying.